2Weekly Mission/Index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="Linkbrarystyle.css" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Abel&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Acme&display=swap"
+      rel="stylesheet"
+    />
+    <title>Linkbrary</title>
+  </head>
+  <body>
+    <header class="gnb">
+      <div class="logoin">
+        <img src="landing/logo.svg" alt="logo" />
+        <a href="./signin.html" class="cta">로그인</a>
+      </div>
+    </header>
+    <main>
+      <div class="content1">
+        <div class="container1">
+          <h2>
+            <span>세상의 모든 정보</span>를<br />
+            쉽게 저장하고 관리해 보세요
+          </h2>
+          <a href="./signup.html" class="cta link">링크 추가하기</a>
+          <div class="image-box1">
+            <img src="./landing/image 25.png" alt="소개 이미지" />
+          </div>
+        </div>
+      </div>
+      <div class="content2">
+        <div class="frame">
+          <div class="ad-text">
+            <h3>
+              <span class="adText-deco1">원하는 링크</span>를<br />
+              저장하세요
+            </h3>
+            <p>
+              나중에 읽고 싶은 글, 다시보고 싶은 영상, 사고 싶은 옷, 기억하고 싶은 모든 것을 한 공간에 저장하세요.
+            </p>
+          </div>
+          <div class="ad-image">
+            <div class="image1"><img src="landing/1번 이미지.PNG" alt="1번이미지"></div>
```

Comment:
1. png 확장자는 소문자로 작성해주세요~! (다른 부분들은 그렇게 되어있네요)
2. 태그마다 줄 바꿈해서 작성해주세요 -> 여기 말고 동일하게 작성된 부분들 같이 수정 부탁드립니다! 
```suggestion
            <div class="image1">
              <img src="landing/1번 이미지.png" alt="1번이미지">
            </div>
```